src/pages/Checklist/index.tsx

```diff
@@ -499,6 +556,73 @@ export const Checklist = () => {
             </li>
           </ul>
         </section>
+        <section>
+          <Heading level={3}>Simulations</Heading>
+          <CheckBox
```

Comment:
I think we should have a description here about what simulations are and why you might want to run them something along the lines of:

```text
The following are are recommended simulations for ensuring your validation setup is robust:
```